src/components/PackageCard/Tags.tsx

```diff
@@ -32,7 +32,7 @@ export const Tags: FunctionComponent = () => {
   return (
     <>
       {tags
-        .slice(0, 3)
+        .slice(0, 10)
```

Comment:
Our package cards currently don't "wrap" the tags area, so if there are 10 tags but only 3 can fit on the first line because of a narrow screen, then only 3 will appear. So this change should be relatively safe / won't cause the cards to become too vertically bloated.